bad.php

```diff
@@ -0,0 +1,6 @@
+<?php
+class SomeClass {
```

Comment:
Each class must be in a namespace of at least one level (a top-level vendor name)
Opening brace of a class must be on the line after the definition